src/Microsoft.ML.CodeGenerator/Microsoft.ML.CodeGenerator.csproj

```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML.AutoML" Version="0.17.0-preview2-28706-1" />
+    <ProjectReference Include="..\Microsoft.ML.AutoML\Microsoft.ML.AutoML.csproj" />
```

Comment:
Sorry to nitpick. Any idea why one of them has a forward slash and the other has a backslash? Can they be the same?

Reply:
I don't know,,, On windows it should be backward slash, but if you check .nupkg file in pkg folder, you can find both type exists

#### Microsoft.ML.AutoML
https://github.com/dotnet/machinelearning/blob/a2406f64c96bb2e5bd7e7c9ba91487e31d673d6b/pkg/Microsoft.ML.AutoML/Microsoft.ML.AutoML.nupkgproj#L9

#### Microsoft.Extensions.ML
https://github.com/dotnet/machinelearning/blob/a2406f64c96bb2e5bd7e7c9ba91487e31d673d6b/pkg/Microsoft.Extensions.ML/Microsoft.Extensions.ML.nupkgproj#L13